### DISCUSSION TITLE
Force refresh/SDG filter issue

### DISCUSSION THREAD

**Discussion by mcrawf35** [getPraxis/FELauncher#7] [Q&A] [answered]
Hello, thanks for this component! It works amazing.  

One issue I am having: when the action payload includes "forcePageRefresh=True", after a user exits the modal, the SDG filters automatically reset.  Ideally, the user-entered filters would stay, and the data displayed in the SDG would still be updated based on changes made in the flow upon exiting the modal.  Any ideas on how to accomplish this? 

Thanks again!
Matt

**Comment by ohadios** [comment, marked as answer]
@mcrawf35 - this would not be possible with the current capabilities of the Enhanced Lightning Grid.
It's a great idea, though, to have that configurable.
We have a project in our sights that may eventually support what you want - but that's probably a little ways out right now.

**Reply by mcrawf35** [reply]
ok, thanks anyways!
